spec/features/ownership_transfer_spec.rb

```diff
@@ -32,32 +32,66 @@
     end
 
     context 'To myself' do
-      pending 'The transfer option is not available' do
-        fail
+      before { transfer_ownership_of_work work, original_owner }
+      it 'Displays an appropriate error message' do
+        expect(page).to have_content 'Sending user must specify another user to receive the work'
       end
     end
 
     context 'To someone else' do
-      pending 'The transfer option is not available' do
-        fail
+      before { transfer_ownership_of_work work, new_owner }
+      it 'Creates a transfer request' do
+        expect(page).to have_content 'Transfer request created'
       end
       context 'If the new owner accepts it' do
-      pending 'The transfer option is not available' do
-        fail
-      end
+        before do
+          new_owner.proxy_deposit_requests.last.transfer!
+          user_utility_toggle.click
+          click_link 'transfer requests'
+        end
+        specify 'I should see it was accepted' do
```

Comment:
Probably the flavor that came along with the copy pasta.  Correcting specify calls to it calls.